Index: src/models/index.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/models/index.ts b/src/models/index.ts
new file mode 100644
--- /dev/null	(date 1702606437197)
+++ b/src/models/index.ts	(date 1702606437197)
@@ -0,0 +1,3 @@
+export * from './common';
+export * from "./user"
+export * from './project'
\ No newline at end of file
